Src/Eiffel/interface/graphical/foundations/tools/es_dockable_tool_panel.e

```diff
@@ -1004,7 +1017,7 @@ feature {NONE} -- Factory
             l_top_padding.set_minimum_height (2)
 
             create l_container
-            l_container.set_minimum_height (26)
+            l_container.set_minimum_height ((create {EV_MONITOR_DPI_DETECTOR_IMP}).scaled_size (26))
```

Comment:
If it is possible to have instance-free call, this would be better, otherwise ok.

Reply:
I will check.